app/templates/agenda-items/index.hbs

```diff
@@ -102,7 +102,6 @@
             @searchField="label"
             @queryParam="gemeentes"
             @placeholder="Alle gemeentes"
-            @value={{this.municipality}}
```

Comment:
The value wasn't used since it is retrieved from the router service.

Reply:
I guess this kind of issues will be caught with Glint template